design-patterns/information/Descriptive ICE/CUBRC p.2

```diff
@@ -0,0 +1,15 @@
+graph BT
```

Comment:
Checked

Reply:
Rename to CUBRC descriptive ICE (p.2)

Reply:
Added more precise names and renamed files